packages/api-client-core/src/support.ts

```diff
@@ -127,11 +129,13 @@ export const assertOperationSuccess = (response: OperationResult<any>, dataPath:
   }
 
   const result = get(response.data, dataPath);
+  const edges = get(result, ["edges"]);
+  const dataArray = edges ?? result;
```

Comment:
In some cases, I was seeing the actual content array nested inside of here, which was not caught by the default `dataPath` that those instances were passing. I felt it would be more robust to test for that centrally (here), than to have to figure out when to add that to the path and when not to.

Reply:
I'm game for that. It's not really something that people use outside of this library, so we can always come back later and adjust.